xtask/src/main.rs

```diff
@@ -0,0 +1,7 @@
+fn main() {
+    if let Err(e) = fastn_xtask::cli() {
```

Comment:
The is still too much. We can improve the error reporting tomorrow if you did not put that here. Make this main function just call the main function from our task. Nothing else. No if, no print. One line only:

```rs
fn main() {
  fastn_xtask::main()
}
```